by-language/java-jooq/jooq.gradle

```diff
@@ -0,0 +1,123 @@
+        classpath 'nu.studer:gradle-jooq-plugin:8.1'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply plugin: nu.studer.gradle.jooq.JooqPlugin
+
+dependencies {
+    jooqGenerator 'org.postgresql:postgresql:42.5.1'
+    jooqGenerator 'org.jooq:jooq-meta-extensions:3.17.7'
+}
+
+jooq {
+
+    // Defaults (can be omitted).
+    // version = '3.17.6'
+    // edition = nu.studer.gradle.jooq.JooqEdition.OSS
+
+    configurations {
+        // Name of the jOOQ configuration.
+        main {
+
+            // Do not *automatically* generate code.
+            generateSchemaSourceOnCompilation = false
+
+            generationTool {
+                logging = org.jooq.meta.jaxb.Logging.WARN
+                generator {
+                    name = 'org.jooq.codegen.DefaultGenerator'
+                    strategy.name = 'org.jooq.codegen.DefaultGeneratorStrategy'
+                    database {
+                        name = 'org.jooq.meta.extensions.ddl.DDLDatabase'
+                        properties {
+
+                            // Specify the location of your SQL script.
+                            // You may use ant-style file matching, e.g. /path/**/to/*.sql
+                            //
+                            // Where:
+                            // - ** matches any directory subtree
+                            // - * matches any number of characters in a directory / file name
+                            // - ? matches a single character in a directory / file name
+                            property {
+                                key = 'scripts'
+                                value = 'src/main/resources/*.sql'
+                            }
+
+                            // The sort order of the scripts within a directory, where:
+                            //
+                            // - semantic: sorts versions, e.g. v-3.10.0 is after v-3.9.0 (default)
+                            // - alphanumeric: sorts strings, e.g. v-3.10.0 is before v-3.9.0
+                            // - flyway: sorts files the same way as flyway does
+                            // - none: doesn't sort directory contents after fetching them from the directory
+                            property {
+                                key = 'sort'
+                                value = 'semantic'
+                            }
+
+                            // The default schema for unqualified objects:
+                            //
+                            // - public: all unqualified objects are located in the PUBLIC (upper case) schema
+                            // - none: all unqualified objects are located in the default schema (default)
+                            //
+                            // This configuration can be overridden with the schema mapping feature
+                            property {
+                                key = 'unqualifiedSchema'
+                                value = 'none'
```

Comment:
I think this confuses a bit as in the examples, no schema is used. I think this works as the `testdrive` is passed into the connection string as the database parameter which CrateDB then uses as the default schema name. Maybe rather use `testdrive` here and not inside the connection string and/or use the schema also in the application?

Reply:
I already tried to use a real value there, but the setting apparently only accepts `public` or `none`.

Reply:
See also https://github.com/crate/cratedb-examples/pull/12#pullrequestreview-1269557974.